internal/tezos/get_receipt.go

```diff
@@ -24,7 +25,6 @@ type receiptExtraInfo struct {
 	Counter             *fftypes.FFBigInt `json:"counter"`
 	Fee                 *fftypes.FFBigInt `json:"fee"`
 	Status              *string           `json:"status"`
-	ErrorMessage        *string           `json:"errorMessage"`
```

Comment:
This error message is used as a part of the transaction receipt, which is needed to be returned to the end users, therefore, this field should remain.